src/plugins/http.js

```diff
@@ -143,7 +144,17 @@ module.exports = [
   },
   {
     name: 'https',
-    patch,
+    patch: (http, tracer, config) => {
+      if (semver.satisfies(process.version, '>=9')) {
+        patch(http, tracer, config)
+      } else {
+        /**
+         * Below Node v9 the `https` module invokes `http.request`, which would end up counting requests twice.
+         * So rather then patch the `https` module, we ensure the `http` module is patched and we count only there.
+         */
+        require('http')
+      }
+    },
     unpatch
   }
```

Comment:
@rochdev Ok this is the simplified version that just checks the Node version and patches the http module instead of the https module. You said in Slack that you think the unpatching part is fine to be left untouched like this?

Reply:
Hold off, I’m investigating some issues on at least Node v10.

Reply:
If I'm not mistaken the signature changed in Node 10 too.

Reply:
Ok, fixed, I forgot that the instrumenter expects the functions to bind to it.

Reply:
Interesting, still appears to be an issue on Node v6. Looking into it.

Reply:
v10 is good btw https://github.com/nodejs/node/blob/v10.10.0/lib/https.js#L280

Reply:
What I meant is that Node 10 accepts 3 parameters while Node 8 accepts only 2:

https://nodejs.org/docs/latest-v10.x/api/http.html#http_http_request_url_options_callback
https://nodejs.org/docs/latest-v8.x/api/http.html#http_http_request_options_callback

This is unrelated and can be done in another PR, but if it's an easy fix it would be great to fix all Node versions at once :)

Reply:
Ah I see. Yeah sure, I can get that done too 👍

Reply:
Done